pkg/build/build.go

```diff
@@ -678,6 +683,19 @@ func (b *Build) populateWorkspace(ctx context.Context, src fs.FS) error {
 		return err
 	}
 
+	// Write out build settings into workspacedir
+	// For now, just the gcc spec file and just link settings.
+	// In the future can control debug symbol generation, march/mtune, etc.
+	specFile, err := os.Create(filepath.Join(b.WorkspaceDir, ".melange.gcc.spec"))
```

Comment:
@jonjohnsonjr i don't like that i have to create this file inside the workspace. I really wish to have it in "/etc/melange/gcc.spec" in the container image that melange builds with apko. And apko can create files, dirs and symlinks.... but it doesn't have ability to write out a text file which would be really nice.

I really wish https://github.com/chainguard-dev/apko/blob/main/docs/apko_file.md#paths would have a key called "contents" with a multi-line string.

Reply:
forking to https://github.com/chainguard-dev/melange/issues/1750 in case this is deemed not easy/quick to implement. As it really is a wishlist.